backend/LexCore/Entities/Organization.cs

```diff
@@ -1,9 +1,19 @@
-﻿namespace LexCore.Entities;
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq.Expressions;
+using EntityFrameworkCore.Projectables;
+
+namespace LexCore.Entities;
 
 public class Organization : EntityBase
 {
     public required string Name { get; set; }
     public required List<OrgMember> Members { get; set; }
+    public required List<Project> Projects { get; set; }
+
+    [NotMapped]
+    [Projectable(UseMemberBody = nameof(SqlProjectCount))]
+    public int ProjectCount { get; set; }
+    private static Expression<Func<Organization, int>> SqlProjectCount => org => org.Projects.Count;
```

Comment:
This will be useful in the org page, for showing the project count as a badge in the tabs right away, while possibly deferring loading the entire list of projects until the Projects tab is clicked on. It will also be useful in the org list page, where we'll want to add a Projects column alongside the existing Members column (both of which will just show a count).